riemann/encoding/cashaddr.py

```diff
@@ -96,24 +125,52 @@ def calculate_checksum(prefix, payload):
 
 
 def verify_checksum(prefix, payload):
+    '''
+    Args:
+        prefix   (str)
+        payload  (list of int)
+    Returns:
+        int
+    '''
     return polymod(prefix_expand(prefix) + payload) == 0
 
 
 def b32decode(inputs):
+    '''
+    Args:
+        inputs (str)
+    Returns:
+        list of int
+    '''
     out = list()
     for letter in inputs:
         out.append(CHARSET.find(letter))
     return out
 
 
 def b32encode(inputs):
+    '''
+    Args:
+        inputs ()
```

Comment:
TODO: fill this in

Reply:
I believe inputs should be a list of int